README.md

```diff
@@ -8,15 +8,6 @@
 
 Implements Ethereum's VM in JS
 
-#### Note on Byzantium Support:
```

Comment:
Why should this be removed?

Reply:
I think it should be reworded stating it always supports the latest, for compatibility with X chain use version Y.

Reply:
I added this some time ago with having in mind just to keep this for the transition period, but I also can readd and reword a bit.

Reply:
@axic I've now updated the text to something more generic now fitting better. If you have some text optimizations/criticism articulate it here, but please approve the PR anyhow, I wouldn't want to run all the tests once again for some text change.

I would the update this on some future occasion on another PR.